ACadSharp/Tables/MultiLeaderStyle.cs

```diff
@@ -0,0 +1,312 @@
+
+		/// <summary>
+		/// Text Angle Type
+		/// </summary>
+		[DxfCodeValue(175)]
+		public TextAngleType TextAngle { get; set; }
+
+		/// <summary>
+		/// Text Alignment Type
+		/// </summary>
+		[DxfCodeValue(176)]
+		public TextAlignmentType TextAlignment { get; set; }
+
+		/// <summary>
+		/// Text Right Attachment Type
+		/// </summary>
+		[DxfCodeValue(178)]
+		public TextAttachmentType TextRightAttachment { get; set; }
+
+		/// <summary>
+		/// Text Color
+		/// </summary>
+		[DxfCodeValue(93)]
+		public Color TextColor { get; set; }
+
+		/// <summary>
+		/// Text Height
+		/// </summary>
+		[DxfCodeValue(45)]
+		public double TextHeight { get; set; }
+
+		/// <summary>
+		/// Enable Frame Text
+		/// </summary>
+		[DxfCodeValue(292)]
+		public bool TextFrame { get; set; }
+
+		/// <summary>
+		/// Text Align Always Left
+		/// </summary>
+		[DxfCodeValue(297)]
+		public bool TextAlignAlwaysLeft { get; set; }
+
+		/// <summary>
+		/// Align Space
+		/// </summary>
+		[DxfCodeValue(46)]
+		public double AlignSpace { get; set; }
+
+		/// <summary>
+		/// Block Content ID
+		/// </summary>
+		[DxfCodeValue(343)]
+		public BlockRecord BlockContent { get; set; }
+
+		/// <summary>
+		/// Block Content Color
+		/// </summary>
+		[DxfCodeValue(94)]
+		public Color BlockContentColor { get; set; }
+
+		/// <summary>
+		/// Block Content Scale
+		/// </summary>
+		[DxfCodeValue(47, 49, 140)]
+		public XYZ BlockContentScale { get; set; }
+
+		/// <summary>
+		/// Enable Block Content Scale
+		/// </summary>
+		[DxfCodeValue(293)]
+		public bool EnableBlockContentScale { get; set; }
+
+		/// <summary>
+		/// Block Content Rotation
+		/// </summary>
+		[DxfCodeValue(141)]
+		public double BlockContentRotation { get; set; }
+
+		/// <summary>
+		/// Enable Block Content Rotation
+		/// </summary>
+		[DxfCodeValue(294)]
+		public bool EnableBlockContentRotation { get; set; }
+
+		/// <summary>
+		/// Block Content Connection Type
+		/// </summary>
+		[DxfCodeValue(177)]
+		public BlockContentConnectionType BlockContentConnection { get; set; }
+
+		/// <summary>
+		/// Scale
+		/// </summary>
+		[DxfCodeValue(142)]
+		public double ScaleFactor { get; set; }
+
+		/// <summary>
+		/// Overwrite Property Value
+		/// B	295	Property changed, meaning not totally clear
+		/// might be set to true if something changed after loading,
+		/// or might be used to trigger updates in dependent MLeaders.
+		/// sequence seems to be different in DXF
+		/// </summary>
+		[DxfCodeValue(295)]
+		public bool OverwritePropertyValue { get; set; }
+
+		/// <summary>
+		/// Is Annotative
+		/// </summary>
+		[DxfCodeValue(296)]
+		public bool IsAnnotative { get; set; }
+
+		/// <summary>
+		/// Break Gap Size
+		/// </summary>
+		[DxfCodeValue(143)]
+		public double BreakGapSize { get; set; }
+
+		/// <summary>
+		/// Text attachment direction for MText contents
+		/// </summary>
+		/// <value>
+		/// 	0 = Horizontal
+		/// 	1 = Vertical
+		/// </value>
```

Comment:
The value tag for the documentation should be only used to add value restrictions like a range of values, enumeration types don't need to be specified.